### DISCUSSION TITLE
FoxESS H1-5.0 Inverter - Export being capped at 4.5 kW

### DISCUSSION THREAD

**Discussion by andrewmooreio** [nathanmarlor/foxess_modbus#593] [Support] [answered]
Hi all,

Not really specific to this integration, but wondered if I could get some input anyway.

Whenever it's sunny and generation from the PVs is good, I noticed my export seems to cap at ~4.5 kW. Any excess above that is put into the batteries, even though the work mode is set to "Feed-in First".

Here's a screenshot of it in action:
![image](https://github.com/nathanmarlor/foxess_modbus/assets/20435317/2e286bc8-f25c-4fd6-b7e7-0ca9e0fa0470)

It's not a huge issue, but I'd rather all of that power is going to the grid, I don't need the battery charging. Is there a setting on the inverter I'm missing?

Thanks in advance!

**Comment by canton7** [comment, marked as answer]
This comes down to what you've negotiated with your DNO. If you're under a standard G98 you're limited to 3.68kW. If you've applied for a higher limit under G99, then the limit will be whatever you've agreed with your DNO.

(DNOs limit the amount you can export to avoid unbalancing the grid)

**Comment by andrewmooreio** [comment]
I see, it hadn't crossed my mind that it could be the DNO limiting. Thanks!

Strangely enough, the installed applied for G99, but it still hasn't been approved. So I would've expected to be limited to 3.68 kW in that case.

**Reply by canton7** [reply]
It's a setting on the inverter which is set by your installer, so this sounds like the installer messed up?

If that's the standard energy flow card, it's well-known for being incorrect in some cases.

**Reply by andrewmooreio** [reply]
I've just checked and the Export Limit on the inverter is set to 60000W which is the default.

I'm using the [power-flow-card-plus](https://github.com/flixlix/power-flow-card-plus) card. The export value comes from the inverter CT so should be accurate, it matches up with what I see in the FoxESS and myenergi apps (Zappi CT).

**Reply by FozzieUK** [reply]
@andrewmooreio you’ll also need to check what ‘safety’ the inverter has been set to - if it’s been set to G98 that would also throttle it.

**Reply by andrewmooreio** [reply]
@FozzieUK where would I find this? Can't see it in the function tree. Thanks.

**Reply by FozzieUK** [reply]
@andrewmooreio you’ll find it in Setting, under ‘On-Grid’, ‘Safety’ - UK would be set to either G98 or G99
